catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -120,7 +124,8 @@ public void init() {
         (req, res) -> {
           String query = req.queryParams("q");
           List<Suggestion> results = this.featureService.getSuggestedFeatureNames(query, 10);
-          return mapper.toJson(results);
+          List<Suggestion> enhanced = coordinateProcessor.enhanceResults(results, query);
```

Comment:
✏️ No need to make a new list here. You can reuse `results` or squash it all into one line.